lib/index.js

```diff
@@ -10,20 +13,38 @@ function createPreprocessor(args, config, logger, helper) {
   var log = logger.create('preprocessor.babel');
   config = config || {};
 
-  function preprocess(content, file, done) {
-    log.debug('Processing "%s".', file.originalPath);
+  var cachePath = config.cachePath;
```

Comment:
Could you make this work with custom processors as well? `var cachePath = args.cachePath || config.cachePath;`